src/script_lua.c

```diff
@@ -1039,6 +1039,61 @@ static void luaRemoveUnsupportedFunctions(lua_State *lua) {
     lua_setglobal(lua,"dofile");
 }
 
+/* Create a global protection function and put it to registry.
+ * This need to be called once in the lua_State lifetime.
+ * After called it is possible to use luaSetGlobalProtection
+ * to set global protection on a give table.
+ *
+ * The function assumes the Lua stack have a least enough
+ * space to push 2 element, its up to the caller to verify
+ * this before calling this function. */
```

Comment:
maybe also refer to `luaEnableGlobalsProtection` and explain the differences?
in which case, maybe it should be below it?

Reply:
Done